src/libraries/Microsoft.PowerFx.Interpreter/EvalVisitor.cs

```diff
@@ -753,7 +753,7 @@ public override async ValueTask<FormulaValue> Visit(RecordFieldAccessNode node,
 
         public override async ValueTask<FormulaValue> Visit(SingleColumnTableAccessNode node, EvalVisitorContext context)
         {
-            return CommonErrors.NotYetImplementedError(node.IRContext, "Single column table access");
+            return CommonErrors.NotYetImplementedError(node.IRContext, CultureInfo, "Single column table access");
```

Comment:
>"Single column table access" [](http://example.com/codeflow?start=84&length=28)

This should be localized (or throw an exception instead of returning this error)